docs/user-guide/projects/project-settings/custom-fields/README.md

```diff
@@ -1,25 +1,25 @@
 ---
 sidebar_navigation:
-  title: Custom fields
+  title: Enable work package custom fields
   priority: 700
 description: Manage custom fields in a project.
 robots: index, follow
-keywords: custom fields
+keywords: custom fields, activate work package custom field
 ---
 # Enable custom fields in projects
 
-**Custom fields can be activated or deactivated in a project**. Open the respective project and go to -> *project settings* -> *custom fields*.
+**Custom fields for work packages can be activated or deactivated in a project**. 
 
-<div class="glossary">**Custom fields** are defined as additional value fields which can be added to existing value fields. The different sections that can use custom fields are work packages, spent time, projects, versions, users, groups, activities (time tracking), and work package priorities.</div>
-1. Manage the custom field by clicking on the name. You have to be a System Administrator in order to [manage custom fields in the system administration](../../../../system-admin-guide/custom-fields).
-2. **Select if the custom fields shall be enabled in the project**. If enabled globally in the custom fields settings, it will automatically be displayed in all projects.
-3. View the work package types for which the custom field is already enabled. Only for the displayed types the custom field will be active.
+<div class="glossary">**Custom fields** are defined as additional attribute fields which can be added to existing attribute fields. The different sections that can use custom fields are work packages, spent time, projects, versions, users, groups, activities (time tracking), and work package priorities.</div>
+
+**Please note:** The instructions in this section *only* apply to custom fields for work packages.
+
+Before you can enable a custom field it needs to be created in the [system administration](../../../../system-admin-guide/custom-fields). Afterwards, open the respective project and go to -> *Project settings* -> *Custom fields*.
+
+1. Manage the custom field by clicking on the name.
+2. **Select if the custom fields shall be enabled in the project**. Some custom field have already been enabled globally for all projects in the system administration.
```

Comment:
Instead of writing "Some custom fields have already been enabled [...]" we could write something along the lines "If custom fields have been enabled globally for all project (in the system administration), [...]". Otherwise, users may think that some custom fields are always enabled globally.
(Small detail - everything else looks good. Thank you.)